fips202/fips202x.h

```diff
@@ -0,0 +1,114 @@
+#ifndef FIPS_202X_H
+#define FIPS_202X_H
+
+#ifndef KECCAK_WAY
+#define KECCAK_WAY 7
```

Comment:
Why 7?

Reply:
Ah, I tested with different number of KECCAK_WAY and 7 is the last number in the code. It works with any KECCAK_WAY.

Reply:
set to 4.